cfme/ansible_tower/explorer.py

```diff
@@ -126,6 +133,20 @@ def is_displayed(self):
         )
 
 
+class OptionForm(View):
+    dialog_name = TextInput('dialog_name')
+
+
+class TowerExplorerJobServiceDialogView(TowerExplorerView):
+    options = View.nested(OptionForm)
+    save_button = Button('Save')
+
+    @property
+    def is_displayed(self):
+        expected_title = 'Adding a new Service Dialog from "{}"'.format(self.context['object'].name)
+        return self.title.text == expected_title
```

Comment:
I would also check `self.in_tower_explorer`... more checks in `is_displayed` always good as our navigation totally depends upon this property.